spec/models/order_spec.rb

```diff
@@ -3,18 +3,23 @@
 require 'date'
 
 RSpec.describe Order do
+  before do
+    @user_id_london = "user id london"
+    @user_id_new_work =  "user id new york"
+    User.create(user_id: @user_id_london, office: "london")
+    User.create(user_id: @user_id_new_work, office: "new york")
+  end
+
   it "only returns order from office requested" do
-    User.create(user_id: "the id", office: "london")
-    User.create(user_id: "bob id", office: "new york")
-    Order.new(
+    Order.place(
       :user_name => "Tom",
-      :user_id => "bob id",
+      :user_id => @user_id_new_work,
       :lunch => "A lunch",
       :date => Days.monday
-    ).save
+    )
     order = Order.new(
```

Comment:
use `Order.place` here, like below?

Reply:
I need the variable for the test, it feels like some abstraction in the tests to make them nicer to read will be welcome!